src/annmas/annotate/command.py

```diff
@@ -14,6 +15,10 @@
 import pysam
 import multiprocessing as mp
 
+import gzip
+from collections import OrderedDict
```

Comment:
Do you actually need this `OrderedDict` here?

Reply:
Oh yeah, good catch - I think I don't anymore.  I had copied it over from some code that does use it and I forgot to remove it.  I'll take care of that.